rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/SchedulerClient.java

```diff
@@ -969,12 +965,14 @@ public void removeThirdPartyCredential(String key) throws NotConnectedException,
 
     @Override
     public Page<TaskState> getTaskStates(String taskTag, long from, long to, boolean mytasks,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Method has 10 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00107)